projects/hr-portal/styles.scss

```diff
@@ -0,0 +1,34 @@
+body {
```

Comment:
Could you clean up this file please? There are styles that you probably dont need